toolkit/scripts/toolkit.mk

```diff
@@ -28,12 +28,12 @@ clean-package-toolkit:
 	rm -rf $(toolkit_build_dir)
 
 package-toolkit: go-tools
-	rm -rf $(toolkit_build_dir) && \
-	mkdir -p $(toolkit_build_dir) && \
-	mkdir -p $(toolkit_tools_dir) && \
-	cp -r $(toolkit_root)/* $(toolkit_build_dir) && \
-	cp $(toolkit_component_extra_files) $(toolkit_build_dir) && \
-	cp $(go_tool_targets) $(toolkit_tools_dir) && \
-	echo "$(toolkit_version)" > $(toolkit_release_file) && \
-	rm -rf $(toolkit_build_dir)/out && \
-	tar -I $(ARCHIVE_TOOL) -cvp -f $(toolkit_archive) -C $(toolkit_build_dir)/.. $(notdir $(toolkit_build_dir))
+	rm -rf "$(toolkit_build_dir)" && \
+	mkdir -p "$(toolkit_build_dir)" && \
+	mkdir -p "$(toolkit_tools_dir)" && \
+	cp -r $(toolkit_root)/* "$(toolkit_build_dir)" && \
+	cp $(toolkit_component_extra_files) "$(toolkit_build_dir)" && \
+	cp $(go_tool_targets) "$(toolkit_tools_dir)" && \
+	echo "$(toolkit_version)" > "$(toolkit_release_file)" && \
+	rm -rf "$(toolkit_build_dir)/out" && \
+	tar -I $(ARCHIVE_TOOL) -cvp -f "$(toolkit_archive)" -C $(toolkit_build_dir)/.. $(notdir $(toolkit_build_dir))
```

Comment:
No quotes around the $(toolkit_build_dir) after -C?

Reply:
Similar question for `$(toolkit_build_dir)`, `$(ARCHIVE_TOOL)` and the left sides of all `cp` commands.